src/main/resources/db/migration/V1_4__create_courthouse_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE moj_courthouse (
```

Comment:
courthouse entity needs to me changed to match:-
src/main/java/uk/gov/hmcts/darts/common/entity/Courthouse.java